src/analysis/complex/upper_half_plane/basic.lean

```diff
@@ -330,7 +330,7 @@ lemma exists_SL2_smul_eq_of_apply_zero_one_ne_zero (g : SL(2, ℝ)) (hc : ↑ₘ
                      ∘ ((+ᵥ) v : ℍ → ℍ) ∘ ((•) u : ℍ → ℍ) :=
 begin
   have h_denom := denom_ne_zero g,
-  obtain ⟨a, b, c, d, h, rfl⟩ := g.fin_two_exists_eq_mk,
+  induction g using matrix.special_linear_group.fin_two_induction with a b c d h_det,
```

Comment:
Any reason for renaming from `h` to `h_det`?

Reply:
This was accidental. I got away with it because of the "feature" of `replace` (used below) which makes it behave like `have` if the hypothesis did not actually exist.

In any case, I've reverted the rename now.